c_src/TLH_absTab.c

```diff
@@ -0,0 +1,225 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <glib.h>
```

Comment:
Certainly, the smaller the better when it comes to preparing binary releases.  I cannot speak on the ease or difficulty of including glib, that's something we'd have test.  If it's written in C and they care about backward compatibility, it could be as simple as adding an extra dependency for the rtb-gateway ebuild.  On the other, if we need to ship the library ourselves, the process would be "make a build, try to run it on a machine, figure out which .so file is currently preventing gateway from starting, add it, and repeat."  In the case of a large library like glib, the number of .so files that we may need to include could be large.

I don't know what people think of [uthash](https://troydhanson.github.io/uthash/), but if it, or another small hashtable implementation, would fit the bill, it could save us much effort down the line.